machine-learning/src/main.py

```diff
@@ -23,12 +23,15 @@ class ClipRequestBody(BaseModel):
     "MACHINE_LEARNING_CLASSIFICATION_MODEL", "microsoft/resnet-50"
 )
 object_model = os.getenv("MACHINE_LEARNING_OBJECT_MODEL", "hustvl/yolos-tiny")
-clip_image_model = os.getenv("MACHINE_LEARNING_CLIP_IMAGE_MODEL", "clip-ViT-B-32")
-clip_text_model = os.getenv("MACHINE_LEARNING_CLIP_TEXT_MODEL", "clip-ViT-B-32")
```

Comment:
I hadn't realized these need to be the same, but that actually makes a lot of sense.

Reply:
Yup! The two models are really just different parts of the same CLIP model.